charts/unpoller/README.md

```diff
@@ -0,0 +1,54 @@
+# unpoller
+
+![Version: 2.11.2](https://img.shields.io/badge/Version-2.11.2-informational?style=flat-square) ![Type: application](https://img.shields.io/badge/Type-application-informational?style=flat-square) ![AppVersion: v2.11.2](https://img.shields.io/badge/AppVersion-v2.11.2-informational?style=flat-square)
```

Comment:
This feels like a quickly-outdated place to put a release version.

Reply:
versioning should be updated automatically in CI (Seperate PR), I created a make target that generates this file (Including the versios), the idea is trigger it during the release process, I haven't yet had a look at your CI to see how it can be integrated.

Reply:
added a bit more docs, let me know if that is enough for this section, I was planning to do a separated PR for the unpoller.com site

Reply:
Thank you for the additional context and documentation. This all seems fine with me. I trust @platinummonkey will help you guide this to a high bar contribution. Good luck!